src/controllers/lrt-escalator-elevator-outages.js

```diff
@@ -58,8 +60,14 @@ module.exports = async function(req, res) {
     let newRows = {
       id,
       created_at: latestTimestamp,
-      outages: JSON.stringify(latestColumnRows),
```

Comment:
I can't merge this in if you remove ingredients. Our process for updating IFTTT would be 
1. Add new ingredients to code
2. Merge to master
3. Update triggers in IFTTT
4. Submit for review
5. Remove old unused code/ingredients
6. Merge to master.

Reply:
I added back the old ingredients along with a comment to remove them once the trigger is updated. The updated trigger should also be in my branch (City of Edmonton - Henry - Escalator/Elevator) on the IFTTT platform. Trigger can be found [here](https://platform.ifttt.com/services/City_of_Edmonton_Henry_LRT_Escalator_Elevator/triggers/lrt_escalator_elevator_outages).